fairscale/optim/oss.py

```diff
@@ -301,20 +301,6 @@ def consolidate_state_dict(self, recipient_rank: int = 0) -> None:
 
                 logging.debug("State from rank %s received", rank)
 
-    def local_state_dict(self) -> dict:
-        """ .. deprecated:: 0.1.5
```

Comment:
been deprecated for a while, I think that this should be safe to remove (and there was not much to do with this anyway)